src/interfaces/api-filing-interface.ts

```diff
@@ -64,6 +64,8 @@ export interface ApiFilingIF {
     consentContinuationOut?: {
       expiry: IsoDatePacific
       orderDetails: string
+      country: string
+      region?:string
```

Comment:
So this is for saving the draft? Does it match the schema?

Reply:
Yes. Here is the schema https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/foreign_jurisdiction.json

Reply:
I'm not really sure but I think it has to be like this:
```javascript
foreignJurisdiction: {
  country: string
  region?: string
}
```

Reply:
LGTM - please test by saving and resuming a draft, and then filing it! 🤞

Reply:
I just tested this and it is failing with this change. Reverting it back as it was correctly saving.